release-notes/8.0/8.0.0/api-diff/Microsoft.NETCore.App/8.0.0_System.Runtime.Intrinsics.Arm.md

```diff
@@ -0,0 +1,1352 @@
+# System.Runtime.Intrinsics.Arm
```

Comment:
System.Runtime.Intrinsics.Arm LGTM

Namely adding the `[ConstantExpected]` attribute, but also a few new APIs covering the multiple register load/store instructions.

CC. @dotnet/area-system-runtime-intrinsics